doc/iterations/2025.2/add_project_elements_selection_to_urls.adoc

```diff
@@ -0,0 +1,89 @@
+= (M) Add project elements selection to URLs
+
+== Problem
+
+Currently, Sirius Web has URLs for projects `+https://example.org/projects/project-id/edit+` and their representations `+https://example.org/projects/project-id/edit/representation-id+`.
+
+But there are no URLs that can be used to designate a selection of *project elements* (_models_ and/or _representations_ and/or _semantic elements_).
+This possibility would be useful for documentation and collaboration purposes.
+
+== Key Result
+
+End users shall be able to create links that include a selection of project elements.
+
+=== Acceptance Criteria
+
+- End users can easily obtain a URL from their current selection of project elements in the workbench.
+- When resolving a URL that specifies a selection, the specified project elements automatically get selected in the workbench.
+
+== Solution
+
+- The current URL will change based on the selection made in the workbench.
+- The identifiers of the selected project elements will be stored in the URL in parameter _selection_.
+- The order of the project element identifiers in the URL reflects the order of the project elements in the selection.
+- A project element may appear several times in the selection.
+  This is for instance the case if several different graphical elements in a representation represent the same semantic element.
+- Upon resolving a URL with parameter _selection_, the workbench selection is set to the specified project elements.
+The resulting behavior may depend largely on how the workbench and its sections decide to implement this behavior.
+- In the default workbench, the selection will be dispatched to the _Representation Editor_ (if opened), or otherwise to the _Explorer_ view.
+- In both the _Explorer_ view and the _Representation Editor_, selecting a semantic element means selecting all the graphical elements whose _target_ is that semantic element.
```

Comment:
No, that's just one part of the behavior it's not the case in all representations and not even in all use cases of existing representations of Sirius Components. It is sometime the case in some scenarios but it's not a behavior set in stone.